### DISCUSSION TITLE
Can't boot Uefi

### DISCUSSION THREAD

**Discussion by SimoneAllegri** [linux-surface/linux-surface#977] [Q&A]
Hello everybody

I installed ubuntu by deleting windows and disabling secure boot ...
I can't log into the uefi (honestly I only got it once and I don't know how I did it).
The key combination does not work. the red line with the open lock (secure boot disabled) probably covers the grub menu.

does anyone have a similar experience?

Thanks in advance

**Comment by NP-chaonay** [comment]
1) what is your surface device
2) to go to surface  uefi do this

Shutdown
Hold  vol up then press power once(while still hold vol up)
Wait until password dialogue or UEFI screen appear

3) it don't cover grub in my SLS, I have SB disabled too btw

**Reply by SimoneAllegri** [reply]
Hey

I just follow Windows instruction: "Press and hold the **volume-down** button on the Surface. While you're doing this, press and release the power button."

vol up boot into uefi correctly

Grazie!!!!

**Reply by NP-chaonay** [reply]
> Hey
> 
> I just follow Windows instruction: "Press and hold the **volume-down** button on the Surface. While you're doing this, press and release the power button."
> 
> vol up boot into uefi correctly
> 
> Grazie!!!!

As I read in ms support manual, it is said volume up btw

Good to know its worked
